tests/components/tolo/test_config_flow.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 from tololib.errors import ResponseTimedOutError
 
-from homeassistant.components.dhcp import IP_ADDRESS, MAC_ADDRESS
+from homeassistant.components import dhcp
```

Comment:
Style question (out of curiosity): Why not `from homeassistant.components.dhcp import DhcpServiceInfo`? E.g., we also use `from tololib.errors import ResponseTimedOutError` and not `from tololib import errors` and then `errors.ResponseTimedOutError` later...

Reply:
That is a styling thing, that we have not standardized, so either way is OK.

Reply:
When multiple DiscoveryInfo are implemented, then it saves quite a few line as you can import `dhcp`/`homekit`/`ssdp`/`zeroconf` in a single line.

Reply:
I see, thanks for clarification!